Userland/Libraries/LibWeb/Layout/InlineLevelIterator.cpp

```diff
@@ -129,6 +129,8 @@ Optional<InlineLevelIterator::Item> InlineLevelIterator::next(float available_wi
             .offset_in_node = chunk.start,
             .length_in_node = chunk.length,
             .width = chunk_width,
+            .padding_start = 2,
+            .padding_end = 2,
```

Comment:
This is incorrect. The padding in InlineLevelIterator items comes from CSS, we can't just hard-code 2px of left and right padding around every text node everywhere :)

To customize the appearance of the text inside `<input>`, I suggest you add some CSS to the shadow DOM inside the `<input>` element. See `HTMLInputElement::create_shadow_tree_if_needed()` where we currently set the style to `white-space:pre`.

Reply:
Agree! I have done as suggested and added this to the CSS in the shadow DOM inside the `<input>` element. Also compared us to the default padding of other browsers. Firefox on Linux and Edge on Windows both use a default padding of top/bottom: 1px and left/right: 2px.

Browsing other web sites like lite.duckduckgo.com, where they set the padding explicitly for the text `<input>` element, correctly overrides our defaults and we behave equally to Firefox and Edge in this scenario also.